jobs/tests.py

```diff
@@ -1,3 +1,28 @@
 from django.test import TestCase
 
-# Create your tests here.
+from model_mommy import mommy
+
+from jobs.models import Job
+
+
+class JobMethodTests(TestCase):
+
+	def setUp(self):
+		self.job = mommy.make(Job, ready_to_publish=True)
+
+	def test_is_published(self):
+		published_date_is_not_none = False
+		self.job.publish()
+		if self.job.published_date:
+			published_date_is_not_none = True
+		self.assertEqual(published_date_is_not_none, True)
+		print "Published date:{0}".format(self.job.published_date)
```

Comment:
I'd suggest that you test the negative case too - create a job that has ready_to_publish False, and test that trying to publish it doesn't succeed.
Also test that what happens if you try to publish a job that's already published (with ready_to_publish True and False)